src/common.speech/AgentConfig.ts

```diff
@@ -28,7 +28,8 @@ export interface IAgentConfig {
     botInfo: {
         commType: string,
         connectionId: string,
-        conversationId: string
+        conversationId: string,
+        fromId: string
```

Comment:
I am not sure where is this used.

Reply:
It's a property that can be set by the customer. WebChat requested it as did the user in this post https://github.com/microsoft/cognitive-services-speech-sdk-js/issues/103